index.html

```diff
@@ -1,312 +1,351 @@
+          </li>
+        </ul>
 
-        <div class="svg2-language">
-          <img src="/image/Rectangle56.svg" alt="rectangle-image" />
-          <ul>
-            <li id="framework">Frameworks</li>
-          </ul>
-        </div>
-        <div>
-          <ul class="js-section2">
-            <li class="js btnn">Bootstrap</li>
-            <li class="rb btnn">Ruby on Rails</li>
-            <li class="rs btnn">RSPec</li>
-            <li class="cp btnn">CapyBara</li>
-            <li class="se btnn">Selenium</li>
-          </ul>
+        <hr class="line2" />
+      </div>
+      <div class="langs">
+        <div class="a1">
+          <div class="svg-language">
+            <img src="/image/Rectangle55.svg" alt="rectangle-image" />
+            <ul>
+              <li id="language">Language</li>
+            </ul>
+          </div>
+          <div>
+            <ul class="js-section">
+              <li class="js btnn">Javascript</li>
+              <li class="rb btnn">Ruby</li>
+              <li class="ht btnn">Html</li>
+              <li class="cs btnn">Css</li>
+            </ul>
+          </div>
         </div>
-
-        <div class="svg3-language">
-          <img src="/image/Rectangle57.svg" alt="rectangle-image" />
-          <ul>
-            <li id="skills-section">Skills</li>
-          </ul>
+        <div class="a2">
+          <div class="svg2-language">
+            <img src="/image/Rectangle56.svg" alt="rectangle-image" />
+            <ul>
+              <li id="framework">Frameworks</li>
+            </ul>
+          </div>
+          <div>
+            <ul class="js-section2">
+              <li class="js btnn">Bootstrap</li>
+              <li class="rb btnn">Ruby on Rails</li>
+              <li class="rs btnn">RSPec</li>
+              <li class="cp btnn">CapyBara</li>
+              <li class="se btnn">Selenium</li>
+            </ul>
+          </div>
         </div>
-        <div class="section3">
-          <ul class="js-section3">
-            <li class="cd btnn">Codekit</li>
-            <li class="gt btnn">Github</li>
-            <li class="cd btnn">Codepen</li>
-            <li class="gt btnn">Gitlab</li>
-            <li class="tr btnn">Terminal</li>
-          </ul>
+        <div class="a3">
+          <div class="svg3-language">
+            <img src="/image/Rectangle57.svg" alt="rectangle-image" />
+            <ul>
+              <li id="skills-section">Skills</li>
+            </ul>
+          </div>
+          <div class="section3">
+            <ul class="js-section3">
+              <li class="cd btnn">Codekit</li>
+              <li class="gt btnn">Github</li>
+              <li class="cd btnn">Codepen</li>
+              <li class="gt btnn">Gitlab</li>
+              <li class="tr btnn">Terminal</li>
+            </ul>
+          </div>
         </div>
-      </section>
-      <section class="form-section">
+      </div>
+    </section>
+    <section class="form-section">
```

Comment:
- [ ] Kindly add more margins or padding to catch breaking changes. see image below
![image](https://user-images.githubusercontent.com/67957533/130256113-068f1d98-209c-41a0-af88-ad0c2dff3ab4.png)